modules/core/src/test/java/org/apache/ignite/internal/processors/service/ServiceHotRedeploymentViaDeploymentSpiTest.java

```diff
@@ -61,12 +66,6 @@
         return cfg;
     }
 
-    /** */
-    @BeforeClass
```

Comment:
I think that we shouldn't remove this check because this test is relevant only for the new Service Grid implementation.
We can remove such checks with the legacy SG-processor in future.

Reply:
New test added to this class `testServiceDeploymentViaDeploymentSpi` which also relevant to legacy implementation, so I've moved this check to `serviceHotRedeploymentTest` to skip legacy implementation only for `testServiceHotRedeploymentNode` and `testServiceHotRedeploymentThinClient`

Reply:
Got it.
Thank you for your clarification.

Reply:
Got it.
Thank you for your clarification.